tests/encodehelp.h

```diff
@@ -101,6 +103,7 @@ static bool process_cmdline(int argc, char *argv[])
         {"refnum", required_argument, NULL, 0 },
         {"idrinterval", required_argument, NULL, 0 },
         {"refmode", required_argument, NULL, 0 },
+        {"bitdepth", required_argument, NULL, 0 },
```

Comment:
Do we really need a extra config for bit depth?Could we just guess bit depth from yuv format or fourcc.